src/selectors/isSaveIndicatorShown.js

```diff
@@ -0,0 +1,6 @@
+import {createSelector} from 'reselect';
+
+export default createSelector(
+  state => state.getIn(['ui', 'saveIndicatorShown']),
+  saveIndicatorShown => saveIndicatorShown,
+);
```

Comment:
So there is no requirement that we use `reselect` for every selector—it’s useful when a selector is doing non-trivial computation, since it’s defined as a two-stage process of “cheaply get some source data and see if it’s changed” followed by “expensively do some calculations on that source data”.

But in this case, there’s no calculation at all! It’s just an identity function.

So, more simply:

```js
export default function isSaveIndicatorShown(state) {
  return state.getIn(['ui', 'saveIndicatorShown']);
}
```